src/ui/public/state_management/state_storage/hashing_store.js

```diff
@@ -50,23 +48,13 @@ export default class HashingStore {
    *  @param {any} the value to hash
    *  @return {string} the hash of the value
    */
-  add(object) {
+  hashAndSetItem(object) {
     const json = angular.toJson(object);
     const hash = this._getShortHash(json);
     this._store.setItem(hash, json);
     return hash;
   }
 
-  /**
-   *  Remove a value identified by the hash from the store
-   *
-   *  @param {string} hash
-   *  @return {undefined}
-   */
-  remove(hash) {
-    this._store.removeItem(hash);
-  }
```

Comment:
I saw this was used for tests, but nothing else. If there's no production use case for this method, I think it's better to remove it and find a way to test this class's functionality without it.